assets/pages/oneoff-contributions/helpers/ajax.js

```diff
@@ -53,13 +53,10 @@ function requestData(paymentToken: string, getState: () => PageState) {
       marketing: state.page.user.gnmMarketing,
       postcode: state.page.user.postcode,
       ophanPageviewId: 'dummy', // todo: correct ophan pageview id
-      intcmp: state.common.intCmp,
+      intcmp: state.common.acquisition.campaignCode,
```

Comment:
It would be good to pass the whole acquisition object through at some point I guess

Reply:
Yeah!

Next step is getting it posted through for regular contributions. Step after that is getting it to work for one-off 🙂

Reply:
Yes we should definitely do that for all the pages 👍